app/src/test/java/org/schabi/newpipe/util/FilePathHelperTest.java

```diff
@@ -0,0 +1,38 @@
+package org.schabi.newpipe.util;
+
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class FilePathHelperTest {
+    @Test
+    public void testIsValidDirectoryPath() throws IOException {
```

Comment:
Please split up the tests into seperate functions. Basically what is expressed with the comments, can be done with the method name. Which then also gets reflected in the test failure description.

Reply:
fixed